### DISCUSSION TITLE
getting problem on output file

### DISCUSSION THREAD

**Discussion by Osiris72** [projectdiscovery/httpx#1332] [Q&A]
![Screenshot at 2023-08-27 19-28-34](https://github.com/projectdiscovery/httpx/assets/141671825/90b32253-3c18-4fe7-b28c-c5e768c76d08)
![Screenshot at 2023-08-27 19-29-03](https://github.com/projectdiscovery/httpx/assets/141671825/1a5c67ba-0884-4059-b9fb-325f6d194109)

**Comment by ehsandeep** [comment]
```
   -nc, -no-color  disable colors in cli output
```

@Osiris72 use this option.

**Reply by Osiris72** [reply]
Thank you
